q2_diversity/_alpha/_visualizer.py

```diff
@@ -0,0 +1,81 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import json
+import os
+import shutil
+
+import scipy
+import numpy
+import qiime
+import pandas as pd
+from statsmodels.sandbox.stats.multicomp import multipletests
+from trender import TRender
+
+
+def alpha_group_significance(output_dir: str, alpha_diversity: pd.Series,
+                             metadata: qiime.Metadata) -> None:
+    metadata_df = metadata.to_dataframe()
+    metadata_df = metadata_df.select_dtypes(exclude=[numpy.number])
```

Comment:
Filter out samples where metadata is empty string in the same way that I'm doing it in `beta_group_significance`:

`metadata = metadata.replace(r'', np.nan).dropna()`